src/components/ChatList/elements/ChannelContent.style.ts

```diff
@@ -9,22 +9,24 @@ export const channelContentStyles = StyleSheet.create({
     fontSize: normalize(14),
     lineHeight: 22,
     flex: 1,
-    paddingBottom: 2
+    paddingBottom: 2,
+    color: COLORS.white2
   },
   description: {
     fontFamily: fonts.inter[400],
     fontSize: normalize(14),
     lineHeight: 22,
     alignSelf: 'center',
     flex: 1,
-    marginRight: 4
+    marginRight: 4,
+    color: COLORS.gray500
   },
   time: {
     fontFamily: fonts.poppins[400],
     fontSize: normalize(14),
     lineHeight: 22,
     marginLeft: 20,
-    color: COLORS.blackgrey,
+    color: COLORS.gray500,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the time text has been changed from `COLORS.blackgrey` to `COLORS.gray500`. Again, ensure that this color change is consistent with the overall theme and accessibility standards.

```diff
-    color: COLORS.blackgrey,
+    color: COLORS.gray500,
     alignSelf: 'flex-start'
```

Overall, these changes seem to be part of a larger effort to update the color scheme of the application. It's important to test these changes thoroughly in different lighting conditions and on various devices to ensure that the new colors improve user experience and don't introduce any usability issues.


<!-- This is an auto-generated comment by OSS CodeRabbit -->